README.md

```diff
@@ -114,7 +124,7 @@ You can run the collection's test suites with the commands:
 
 The current process for publishing new versions of the Windows Core Collection is manual, and requires a user who has access to the `ansible` namespace on Ansible Galaxy and Automation Hub to publish the build artifact.
 
-  1. Ensure `CHANGELOG.md` contains all the latest changes.
+  1. Regenerate `CHANGELOG.rst` with the [antsibull-changelog generator](https://github.com/ansible-community/antsibull-changelog).
```

Comment:
I just recently updated the README for this stuff in https://github.com/ansible-collections/ansible.windows/pull/70 which already has something like this. I copied whatever was on the community.crypto repo.

Reply:
kewl thanks!